### ISSUE TITLE
Adapt for Steam Workshop?

### ISSUE THREAD

**Issue by alexandria-p** [RugbugRedfern/Mycelium-Networking-For-Content-Warning#9]
Would you consider publishing Mycelium Networking for CW on the Steam Workshop for Content Warning?

**Comment by RugbugRedfern** [comment]
Yes, that sounds like a good idea. I'll start working on it when I have some free time.
